scripts/info.py

```diff
@@ -0,0 +1,118 @@
+import argparse
+import os
+from pathlib import Path
+from typing import List
+from datasets import load_dataset_builder
+import sys
+
+INFO_KEYS = ['citation', 'description', 'homepage', 'license', 'version', 'acknowledgements']
+
+def get_all_datasets() -> List[str]:
+    # Collect all unique dataset names from the scripts directory
+    all_datasets = set()
+    # only consider directories in the scripts directory
+    for file in os.listdir('scripts'):
+        if os.path.isdir(os.path.join('scripts', file)):
+            all_datasets.add(file)
+    return sorted(list(all_datasets))
+
+def get_info(dataset: str, info_keys: List[str] = ['citation']) -> dict:
```

Comment:
What is the purpose of this function? Can we comment it in a docstring?

Reply:
Updated doc strings.